pages/index.html

```diff
@@ -156,9 +156,19 @@ <h2 class="bannerbox__title">I need a ride</h2>
                             <input type="tel" class="form-input" id="riderPhone" placeholder="Phone" name="RiderPhone" required />
                             <div class="help-block with-errors"></div>
                         </div>
+                        <div class="form-group radio">
+                            <p>Is this a cell phone?</p>
+                            <label>
+                                <input type="radio" name="riderCell" value="Yes">Yes
+                            </label>
+                            <label>
+                                <input type="radio" name="riderCell" value="No">No
+                            </label>
+                        </div>
                         <div class="form-group checkbox checkbox--multi">
-                            <p>Preferred method of contact</p>
-                            <div>How would you like to driver to get in touch with you?</div>
+                            <p>Preferred method of contact from driver (choose as many as you like).</p>
+                            <div>If you select a cell phone, we'll also send you text message updates</div>
+                            <!-- <div>How would you like to driver to get in touch with you?</div> -->
```

Comment:
Please delete this commented line of code, there's no need to leave it around when we can use git to find it again if we need it :)